corehq/apps/app_manager/detail_screen.py

```diff
@@ -321,8 +322,29 @@ def variables(self):
 @register_format_type('enum-image')
 class EnumImage(Enum):
     template_form = 'image'
-    header_width = '13%'
-    template_width = '13%'
+
+    @property
+    def header_width(self):
+        return self.template_width
+
+    @property
+    def template_width(self):
+        '''
+        Set column width to accommodate widest image.
+        '''
+        width = 0
+        if self.app.enable_case_list_icon_dynamic_width:
+            for i, item in enumerate(self.column.enum):
+                for path in item.value.values():
+                    map_item = self.app.multimedia_map[path]
+                    if map_item is not None:
+                        image = CommCareMultimedia.get(map_item.multimedia_id)
+                        if image is not None:
+                            for media in image.aux_media:
+                                width = max(width, media.media_meta['size']['width'])
+        if width == 0:
+            return '13%'
+        return str(width)
```

Comment:
no, media_meta has the pixel dimensions. mobile accepts "X%" or "X" and assumes that the latter is a number of pixels.